test/views/agent/ChargeSummaryViewSpec.scala

```diff
@@ -213,6 +331,12 @@ class ChargeSummaryViewSpec extends TestSupport with FeatureSwitching with ViewS
                 "6 Jul 2018 Remaining balance reduced due to amended return £12,345.00",
                 "12 Aug 2019 Remaining balance reduced by taxpayer request £54,321.00")
             }
+            "Amended balancing charge" in new Setup(documentDetailAmendedBalCharge, chargeHistoryOpt = Some(fullChargeHistory)) {
```

Comment:
This test case `"Amended balancing charge"` duplicates the previous test case `"balancing charge"`.
I would suggest rework one of the test cases to use the `documentDetailBalancingCharge` which has a document description `"TRM New Charge"`.

Reply:
Done